src/agents/trpo/train.py

```diff
@@ -0,0 +1,166 @@
+"""
+Training script for the TD3 agent.
+"""
+
+import torch
+import os
+import wandb
+
+from bipedal_walker.environment import BipedalWalkerEnv
+from agents.trpo.agent import TRPOAgent
+
+from gym.wrappers.record_video import RecordVideo
+
+
+def train_agent(hardcore: bool, render: bool):
+    """
+    Trains the TRPO agent.
+    """
+
+    num_episodes = 3000
+
+    # Initialize WandB
+    wandb.init(
+        project="bipedal-walker",
+        config={
+            "algorithm": "TRPO",
+            "environment": "BipedalWalker-v3",
+            "hardcore": hardcore,
+            "num_episodes": num_episodes,
+            "gamma": 0.99,
+            "max_kl": 0.005,
+            "damping": 0.1,
+            "entropy_coef": 0.01,
+            "value_lr": 1e-3,
+            "max_step_size": 0.01,
+            "backtrack_coeff": 0.5,
+            "l2_reg": 0.01,
+            "network_size": [64, 64]
+        }
+    )
+
+    # Create environment
+    base_env = BipedalWalkerEnv(hardcore, render)
+
+    # Create output dir for videos
+    video_dir = "videos/trpo"
+    os.makedirs(video_dir, exist_ok=True)
+
+    # Setup video recording
+    env = RecordVideo(
+        base_env.env,
+        video_dir,
+        episode_trigger=lambda ep: ep % 1000 == 0,
```

Comment:
Might be an idea to record a few more videos

```suggestion
        episode_trigger=lambda ep: ep % 100 == 0,
```